sql-update/murica.sql

```diff
@@ -0,0 +1,4 @@
+update "countries"
+      set "name" = 'Murica'
+where "countryId" = 103
```

Comment:
Normally, you wouldn't know the id of the row with the country name "United States" up-front. Please filter by the information that you _do_ know up-front:

---

![image](https://user-images.githubusercontent.com/16143771/179885189-62269adc-3dff-4c9d-ad81-93544d973ffe.png)